linode_api4/objects/networking.py

```diff
@@ -56,8 +92,37 @@ def to(self, linode):
 
         if not isinstance(linode, Instance):
             raise ValueError("IP Address can only be assigned to a Linode!")
+
         return {"address": self.address, "linode_id": linode.id}
 
+    def ip_addresses_share(self, ips, linode_id):
+        """
+        Configure shared IPs.
+        """
+
+        params = {
```

Comment:
Just a small UX suggestion, but maybe we could optionally accept a list of `IPAddress` objects here similar to what we do [here](https://github.com/linode/linode_api4-python/blob/dev/linode_api4/linode_client.py#L670).